.github/workflows/gitleaks.yml

```diff
@@ -0,0 +1,28 @@
+#name: Github Secrets Scanner
+
+on: [push]
+
+jobs:
+  build:
```

Comment:
Can we rename this job? It makes it confusing when you're setting up branch protection status checks